.github/codeql/codeql-configuration.yml

```diff
@@ -1,4 +1,6 @@
 name : CodeQL Configuration
 
 paths:
-  - './src' 
+  - src
```

Comment:
Should we add the `scripts` directory?

Reply:
Maybe, though that's pure dev dependency; I actually considered ignoring tests too but opted not to.

Reply:
Added it and the gulpfile; we'll see if it breaks.

Reply:
Looks like quite a few things showed up; the ones about uncontrolled user input are probably ignorable since that's what those scripts intend to do anyway. The rest are sort of interesting.

Reply:
One of these actually looks like a bug; the localization script only replaced the first `]` when it should have replaced all of them. Of course, what we should do is not produce XML via string templates, but, that's another story. Maybe after modules as my stack includes a build change on that front.